integration_tests/suite/base/test_users.py

```diff
@@ -2417,3 +2417,59 @@ def test_update_incall_new_incall():
         ),
     )
     url.delete(recursive=True)
+
+
+@fixtures.device()
+@fixtures.device()
+def test_post_lines_same_unallocated_device_no_error(device, device2):
+    user_resources = generate_user_resources_bodies(context_name=CONTEXT, device=device)
+
+    # define another line with the same extension
+    line2 = {
+        'context': CONTEXT,
+        'extensions': [user_resources.extension],
+        'endpoint_sip': {'name': s.random_string(5)},
+    }
+    line2['device_id'] = device2['id']
+
+    # user creation with 2 lines
+    response = confd.users.post(
+        {
+            'lines': [user_resources.line, line2],
+            **user_resources.user,
+        }
+    )
```

Comment:
Writing the code differently avoid to write comment. Example, if you want to focus on your lines, you can separate this on a specific variable
```suggestion
    lines = [user_resources.line, line2]
    response = confd.users.post(lines=lines, **user_resources.user)
```
**Note: Not 100% if what inside of `user_resources.user`  is compatible with the kwargs, if not, you can re-embedded into a dictionary as before

Reply:
comment removed 
and dict param removed for post(...)